roles/galaxy_post_install/tasks/setup_galaxy_api_access_log.yml

```diff
@@ -7,6 +7,8 @@
       owner: '{{ pulp_user }}'
       group: '{{ pulp_group }}'
       mode: "u+rw"
+    tags:
+      - molecule-idempotence-notest
```

Comment:
```suggestion
      modification_time: preserve
````

This makes the task idempotent.

I haven't verified, but theoretically it will.

Reply:
Nice! I'll test it